manticore/native/memory.py

```diff
@@ -892,6 +892,19 @@ def perms(self, index):
         else:
             return self.map_containing(index).perms
 
+    def max_exec_size(self, index, size):
+        r = 0
```

Comment:
Maybe:
```suggestion
        exec_offset = 0
```
and later on

Reply:
I replaced it with `max_size`